src/common/src/buffer/bitmap.rs

```diff
@@ -804,16 +894,6 @@ mod tests {
         assert_eq!(Bitmap::from_bytes(&[0b01101110]), (bitmap1 | bitmap2));
     }
 
-    #[test]
-    fn test_bitwise_saturate_subtract() {
-        let bitmap1 = Bitmap::from_bytes(&[0b01101010]);
-        let bitmap2 = Bitmap::from_bytes(&[0b01001110]);
-        assert_eq!(
-            Bitmap::from_bytes(&[0b00100000]),
-            Bitmap::bit_saturate_subtract(&bitmap1, &bitmap2)
-        );
-    }
```

Comment:
Why remove this test?

Reply:
I found there is no usage of `Bitmap::bit_saturate_subtract` any more. So I removed this function along with the test.